### PR TITLE
[server][Zabbix] Zabbix macro replacement

### PR DESCRIPTION
**Pull request by cosmo0920**
This pull request a part of works related to issue #326.

Contains:

* get extendDescription  request to ZabbixAPI
* merge extendDescription into plain information which is gotten ZabbixAPI
* save merged information into hatohol.triggers entended_info column

Not yet:

* deliver entended_info column information to hatohol client

### CODE REVIEW

File: server/src/HatoholDBUtils.cc [hunk 1]
```diff
@@ -85,6 +85,56 @@ void HatoholDBUtils::transformTriggersToHatoholFormat(
...
+
+		trigGroupStream.seek(ITEM_ID_ZBX_TRIGGERS_TRIGGERID);
+		trigGroupStream >> trigInfo.id;
+
+		trigGroupStream.seek(ITEM_ID_ZBX_TRIGGERS_EXPANDED_DESCRIPTION);
+		trigGroupStream >> extendedInfo;
+
+		JSONBuilder agent;
+		agent.startObject();
+		agent.add("expanded_description", extendedInfo);
```
Comment (ashie) [inline server/src/HatoholDBUtils.cc]: We use camelcase for key names in JSON.

File: server/common/ZabbixAPI.cc [hunk 2]
```diff
@@ -636,6 +678,31 @@ SoupMessage *ZabbixAPI::queryTrigger(HatoholError &queryRet, int requestSince)
...
+	agent.add("method", "trigger.get");
+
+	agent.startObject("params");
+	agent.startArray("output");
+	agent.add("extend");
+	agent.add("description");
+	agent.endArray();
+	if (requestSince > 0)
+		agent.add("lastChangeSince", requestSince);
+	agent.add("expandDescription", 1);
```
Comment (ashie) [inline server/common/ZabbixAPI.cc]: Please replace "expand" with "expanded".

Comment (cosmo0920) [inline server/common/ZabbixAPI.cc]: Here is requesting Zabbix API code! :confounded:

Comment (ashie) [inline server/common/ZabbixAPI.cc]: Oops, sorry.

**Comment by ashie** [comment]
Confirmed:

* We discussed the design at F2F.
  * Store extended trigger information to a new column (in JSON format)
* Can build it on my environment (Ubuntu 14.04 and Scientific Linux release 6.3)
* Can update DBTablesMonitoring's vresion from 9 to 10 automatically by launching Hatohol server
* Can show triggers page & events page (it doesn't destroy existing features)
* Tests passed on Travis CI

**Comment by cosmo0920** [comment]
Thank you for merging!

File: server/src/DBTablesMonitoring.cc [hunk 3]
```diff
@@ -153,6 +153,15 @@ static const ColumnDef COLUMN_DEF_TRIGGERS[] = {
 	SQL_KEY_NONE,                      // keyType
 	0,                                 // flags
 	NULL,                              // defaultValue
+}, {
+	"entended_info",                   // columnName
```
Comment (bynnchapu) [inline server/src/DBTablesMonitoring.cc]: This typo is fixed at 4869154 revision in #960.

**Pull request merged**